src/main/scala/cromwell/engine/workflow/SingleWorkflowRunnerActor.scala

```diff
@@ -112,8 +119,11 @@ case class SingleWorkflowRunnerActor(source: WorkflowSourceFiles,
       issueReply using updatedData
   }
 
+  var isDone = false
+
   when (Done) {
     case Event(IssueReply, data) =>
+      isDone = true
```

Comment:
I just checked - the SingleWorkflowRunnerActor is what you get when the command line is used to run a ... single workflow. So I think in this case you should be OK. This actor isn't used in server mode - I assume that's what was intended?